src/Sensor/Measurement.java

```diff
@@ -0,0 +1,24 @@
+package Sensor;
+
+import org.hyperic.sigar.Sigar;
+
+
+abstract public class Measurement {
+
+    /**
+     * Type of measurement, set by inheriting classes, readonly for user
+     */
+    private String MeasurementType;
```

Comment:
Use one convention - all variables should or shouldn't start with capital letter